hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -627,6 +672,128 @@ private HoodieInstant fetchNextCompactionInstant() throws InterruptedException {
   }
 
   public DeltaSyncService getDeltaSyncService() {
-    return deltaSyncService;
+    return deltaSyncService.get();
+  }
+
+  /**
+   * Performs bootstrap from a non-hudi source.
+   */
+  public static class BootstrapExecutor  implements Serializable {
```

Comment:
Do you think it's better to relocate this class as a separate class? Like [HDFSParquetImporter](https://github.com/apache/hudi/blob/master/hudi-utilities/src/main/java/org/apache/hudi/utilities/HDFSParquetImporter.java), [HoodieCompactor](https://github.com/apache/hudi/blob/master/hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieCompactor.java)?

Reply:
moved. I was mulling the same.